app/routes/dashboards/dashboard.js

```diff
@@ -534,30 +534,30 @@ export default Ember.Route.extend({
                     },
                 ]
             },
-            resultsList: {
+            search: {
```

Comment:
I think we're deleting the contributors widget right?
and I thought I did object detail this morning. I'll double check